antarest/study/web/output_blueprint.py

```diff
@@ -553,12 +555,12 @@ def get_output_variables_view(
     ) -> Response:
         """
         Fetches the variables view for a given output and a given configuration.
-        If the view does not exist in DB yet, raises an HTTP 404 error.
+        If the view does not exist in DB yet, returns an HTTP 404 response.
```

Comment:
Would be nice to describe those responses in the swagger doc, see:
https://fastapi.tiangolo.com/advanced/additional-responses/#additional-response-with-model